master_changes.md

```diff
@@ -36,6 +36,7 @@ users)
 ## Init
   * Run the sandbox check in the temporary directory [#4787 @dra27 - fix #4783]
   * [BUG] Fix `opam init` and `opam init --reinit` when the `jobs` variable has been set in the opamrc or the current config. [#5056 @rjbou]
+  * Use `.opam` from `%HOME%` or `%USERPROFILE%` on Windows, only if found; otherwise use `%LOCALAPPDATA%\opam` as root. [#5212 @dra27]
```

Comment:
Small nit:

Reply:
```suggestion
  * Use `.opam` from `%HOME%` or `opam` from `%USERPROFILE%` on Windows, only if found; otherwise use `%LOCALAPPDATA%\opam` as root. [#5212 @dra27]
```

Reply:
What I've put is correct inasmuch as it's what it's doing - `%USERPROFILE%` is being used as the fallback if `%HOME%` isn't set - it's only in `%LOCALAPPDATA%` that I'm using `opam` rather than `.opam`?

Reply:
Oops. I mixed up the two. Cancel the suggestion!